src/tests/ftest/launch.py

```diff
@@ -956,13 +957,13 @@ def archive_files(destination, host_list, source_files):
     #
     # pylint: disable=bad-continuation
     commands = [
-        "set -eu",
+        "set -ux",
         "rc=0",
         "copied=()",
-        "for file in $(ls {})".format(source_files),
+        "for file in $(ls -d {})".format(source_files),
```

Comment:
We shouldn't be expecting any directories, correct?

Reply:
They are **_usually_** plain text files, [except for some tests launched via orterun in the CaRT tests which log to test directories](https://github.com/daos-stack/daos/pull/3139#discussion_r496881627).